.changeset/nervous-bags-clap.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-frontend/application-components': patch
+'@commercetools-frontend/application-shell': patch
+---
+
+Move component to application-shell package
```

Comment:
NIT: It will be nice to know the component(s) that were moved. 

```suggestion
Move CustomViewLoader to application-shell package
```

Reply:
Thanks https://github.com/commercetools/merchant-center-application-kit/pull/3175/commits/0f94a4aff80ab993378a8e76bc285060aba0cbe9